### ISSUE TITLE
High CPU usage during long periods of use

### ISSUE THREAD

**Issue by jxxnnee** [shogo4405/HaishinKit.swift#837]
**High CPU usage during long periods of use**

When I first start publishing,CPU usage keep it around 30%.
but CPU usage increases to 70 to 80 percent if I keep publishing for a long time.(About 30 minutes)

**Smartphone (please complete the following information):**
 - Device: iPhone 6+
 - OS: iOS 12.4.8

**i did set capture, video, audio for stream**

```
self.stream.captureSettings = [
            .fps: 30,
            .sessionPreset: AVCaptureSession.Preset.medium,
            .isVideoMirrored: true,
            .preferredVideoStabilizationMode: true,
]
self.stream.videoSettings =[
            .muted: false,
            .width: 720, 
            .height: 1280, 
            .bitrate: 5000 * 1024, 
            .profileLevel: kVTProfileLevel_H264_Baseline_AutoLevel,
            .maxKeyFrameIntervalDuration: 5, // key frame / sec
]
self.stream.audioSettings =[
            .bitrate: 128 * 1000
]
```

**And, Connection Bytes Transmitted per Second**

current FPS:  30
connection.currentBytesOutPerSecond:  345149
connection.currentBytesInPerSecond:  0

**Comment by tatuanfpt** [comment]
Facing same issue when use the sample video + audio setting for screen live streaming to Facebook's RTMP.
Device iPhone Xs Max, iOS 14
<img width="1440" alt="Screen Shot 2021-04-10 at 1 34 02 AM" src="https://user-images.githubusercontent.com/81206187/114226276-98f46200-999d-11eb-9903-188992bf08ab.png">

**Comment by shogo4405** [comment]
Might be fixed it #851.
